bool Bool
int8 Int8
uint8 UInt8
int16 Int16
uint16 UInt16
int32 Int32
uint32 UInt32
int64 Int64
uint64 UInt64
float32 Float32
float64 Float64
string String
time Time
duration Duration
